examples/vite-demo-vanilla-bundle/src/examples/example22.ts

```diff
@@ -8,6 +8,8 @@ import {
 import { SlickCustomTooltip } from '@slickgrid-universal/custom-tooltip-plugin';
 import { Slicker, type SlickVanillaGridBundle } from '@slickgrid-universal/vanilla-bundle';
 import { ExampleGridOptions } from './example-grid-options';
+import { BindingEventService } from '@slickgrid-universal/binding';
```

Comment:
I like separating external vs internal imports, so would you mind moving this import further to the top? Thanks :)

Reply:
how about a linter rule for that? ;)

Reply:
oh does that actually exists? Dang ESLint is so powerful :P

Reply:
I'll create an additional PR after this one as it currently would involve 319 problems ;)

Reply:
its not only separation but also unifies blank lines between those and ordered imports. So with the extra PR you get a chance to tweak those configs as you prefer.

I'll set it up tomorrow as I need to get some rest today